src/scenes/Organizations.re

```diff
@@ -15,66 +16,139 @@ module Styles = {
-        style=Style.(style([paddingHorizontal(Pt(20.))]))
-      />
-      <AppList
-        renderItems={_ =>
-          <ScrollView>
-            <ListItem
-              onPress=ignore
-              renderLeft={_ =>
-                <Title value={j|Itaú|j} style=Styles.contentTitle />
-              }
-              renderCenter={_ =>
-                <View style=Styles.content>
-                  <AppText value="123 members" style=Styles.contentDetail />
-                  <AppText value="4 projects" style=Styles.contentDetail />
-                </View>
-              }
-              renderRight={_ => <ListItemMoreInfo direction=Right />}
-            />
-          </ScrollView>
-        }
-      />
-      <AppFAB
-        icon={
-          Element(
-            _ =>
-              <RNIcons.MaterialCommunityIcons
-                name=`_plus
-                size=24.
-                color={AppConfig.theme.contrastWhite}
-              />,
-          )
-        }
-        small=true
-      />
-      <SectionTitle
-        value="Version"
-        style=Style.(style([paddingHorizontal(Pt(20.))]))
-      />
-      <SectionTitle
-        value="Helper"
-        style=Style.(style([paddingHorizontal(Pt(20.))]))
-      />
-      <SectionTitle
-        value="Whatever"
-        style=Style.(style([paddingHorizontal(Pt(20.))]))
-      />
-    </View>,
+  render: _self => {
+    let%Epitath {result, refetch} =
+      c => <QueryContainer> ...c </QueryContainer>;
+
+    <StackNavigator.Screen navigation>
+      ...{() =>
+        <QueryRenderer result refetch>
+          ...{data =>
+            <View style=Styles.wrapper>
+              <AppStatusBar mode=Light />
+              <AppBar
+                title="Settings"
+                renderLeft={_ => <BackButton onPress=ignore />}
+              />
+              <SectionTitle
+                value="Organizations"
+                style=Style.(style([paddingHorizontal(Pt(20.))]))
+              />
+              <AppList>
+                {data##currentUser
+                 ->Belt.Option.map(user => user##notifications)
+                 ->(
+                     notifications =>
+                       switch (notifications) {
+                       | Some(notifications) =>
+                         Belt.Option.getWithDefault(notifications, [||])
+                         ->Belt.Array.map(organization =>
+                             <ListItem
+                               key=organization##id
+                               onPress=ignore
+                               renderLeft={_ =>
+                                 <Title
+                                   value={j|Itaú|j}
+                                   style=Styles.contentTitle
+                                 />
+                               }
+                               renderCenter={_ =>
+                                 <View style=Styles.content>
+                                   <AppText
```

Comment:
can already hook the title and etc right?

Reply:
yes, but it needs the query for it

Reply:
🤔 thought was already being queried here